invesalius/data/elfin.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+
+import sys
+from time import sleep
+from socket import socket, AF_INET, SOCK_DGRAM, SOCK_STREAM
+import invesalius.constants as const
+
+class elfin_server():
```

Comment:
nit: Minor style issue, but could you name the class using CapWords convention, that is, name it as ElfinServer? Thanks!